src/6-distributions/index.md

```diff
@@ -0,0 +1,60 @@
+---
+date: 2016-01-29
+id: 6-distributions
+template: tmpl/layouts/post.html
+baseurl: ..
+breadcrumbs:
+  - {name: "6-distributions", link: "./" }
+tags: dist, distributions
+title: IPFS distributions
+author: Richard Littauer
+collection: posts
+---
+
+[![](!img/screenshot.png)](http://dist.ipfs.io/)
+
+[dist.ipfs.io](http://dist.ipfs.io/) is the new distributions page for IPFS. This is the new one-stop-shop for finding and downloading all official binaries that IPFS produces.
+
```

Comment:
This is missing a section, before `Project Details`, explaining how this website itself works. The really cool thing about this site is that everything is shipped with ipfs itself, the site uses dnslink, etc. Users can view the site in their local gateways just fine, too: `/ipns/dist.ipfs.io`. This means a bunch of things, like: dist.ipfs.io can be replicated and used to distribute ipfs in networks not connected to the internet.